linux/preview/examples/mssql-mlservices/README.md

```diff
@@ -11,16 +11,30 @@ docker build -t mssql-server-mlservices .
 > **Note:**
 > mssql-server-mlservices is just a suggested name for the container image.  You can use a different name if you want to.
 
+> **Note:**
+> The Dockerfile does not install the mssql-tools (e.g. sqlcmd) - to add them to the dockerfile or install on the host, see https://docs.microsoft.com/en-us/sql/tools/sqlcmd-utility?view=sql-server-ver15
+
+
 ## Run
 1. Once you have built the container image, you can run it by running the following command:
 ```
-docker run -d -e MSSQL_PID=Developer -e ACCEPT_EULA=Y -e ACCEPT_EULA_ML=Y -e SA_PASSWORD=<some password> -v <some directory on the host OS>:/var/opt/mssql -p 1433:1433 mssql-server-mlservices
+docker run -d \
+    -e MSSQL_PID=Developer \
+    -e ACCEPT_EULA=Y \
+    -e ACCEPT_EULA_ML=Y \
+    -e MSSQL_SA_PASSWORD=<some password> \    # !! fixed from original instructions !!
```

Comment:
I appreciate the comment and that you found what was breaking, but adding the comment in line here breaks the copy/paste ! I would just keep that in the comments of the merge request.

Reply:
Yes, sorry - I added that later.